internal/pkg/agent/application/monitoring/process.go

```diff
@@ -61,3 +93,76 @@ func processHandler(coord *coordinator.Coordinator, statsHandler func(http.Respo
 		return errorWithStatus(http.StatusNotFound, fmt.Errorf("matching component %v not found", id))
 	}
 }
+
+func isProcessRedirectable(processName string) bool {
```

Comment:
[Suggestion]
I don't want to be pedantic, but I see it as a bit inconsistent. `vars[processIDKey]` is used to extract the _process ID_, but now it's called _name_. I'd suggest to keep it consistent, using one or the other.